src/pages/home/report/ReportActionContextMenu.js

```diff
@@ -148,10 +159,30 @@ class ReportActionContextMenu extends React.Component {
         return lodashGet(message, 'text', '');
     }
 
+    /**
+     * Can the current user edit this report action?
+     *
+     * @return {Boolean}
+     */
+    canEdit() {
+        return this.props.reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.ADDCOMMENT
+        && this.props.reportAction.actorEmail === this.props.session.email
+        && this.props.reportAction.reportActionID;
```

Comment:
Is there a legitimate case for an action not having `reportActionID`?

Reply:
When it's been not yet confirmed by server, optimistic response. @yuwenmemon might have a bit more info on this.

Reply:
OK, if that's true, would you mind adding it as a comment in the code so that someone doesn't come along and decide to remove it?

Reply:
absolutely